.storybook/mocks.ts

```diff
@@ -0,0 +1,69 @@
+import type { CurrencyType, LanguageType, StoreResponseType } from "@/core/api/graphql/types";
```

Comment:
separate pull request

Reply:
These mocks are required for
https://github.com/VirtoCommerce/vc-frontend/blob/f7fce67749006daffb78352e7545f77ec7dc21d5/.storybook/preview.ts#L17
which is part of bug fix (otherwise it will throw exception "No theme context was found")